lib/tasks/ffmpeg.rake

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
```

Comment:
I would leave this out. This will be the default in ruby 3 and there are no strings in this file.

Reply:
It wasn't included in ruby 3 [after all](https://bugs.ruby-lang.org/issues/11473#note-53) but fair point nonetheless